.taq/config.json

```diff
@@ -85,5 +67,26 @@
         "john": "3_000_000_000",
         "jane": "3_000_000_000",
         "joe": "3_000_000_000"
-    }
+    },
+    "plugins": [
+        {
+            "name": "@taqueria/plugin-ligo",
+            "type": "npm"
+        },
+        {
+            "name": "@taqueria/plugin-flextesa",
+            "type": "npm"
+        },
+        {
+            "name": "@taqueria/plugin-taquito",
+            "type": "npm"
+        },
+        {
+            "name": "@taqueria/plugin-ipfs-pinata",
+            "type": "npm"
+        }
+    ],
+    "projectDir": "/Users/mweichert/Projects/taco-shop",
```

Comment:
Why do we have these 3 new fields? And projectDir and configFile are both machine specific.

Reply:
I'm not sure to be honest. I don't recall adding them. I'll remove them.